cmd/ci-operator/main.go

```diff
@@ -701,6 +714,42 @@ func overrideMultiStageParams(o *options) error {
 	return nil
 }
 
+func overrideTestStepDependencyParams(o *options) error {
+	if len(o.dependencyOverrides.values) > 0 {
```

Comment:
Consider using a guard clause and fast return, like in `overrideMultiStageParams`